template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,18 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
+      // Give paths to parse for rules. These should be absolute! 
+        purifyOptions: {
+            whitelist: ['someclass']
```

Comment:
It possible plugin accidentally will not include some css class to resulted css. So whitelist option contains name of classes those will be included forcibly.

> whitelist Array of selectors to always leave in. Ex. ['button-active', '*modal*'] this will leave any selector that includes modal in it and selectors that match button-active. (wrapping the string with *'s, leaves all selectors that include it)

More detailed description about this plugin
https://github.com/purifycss/purifycss
https://www.npmjs.com/package/purifycss-webpack